manuscript/11-Promises.md

```diff
@@ -71,31 +71,31 @@ In this code, a successful call to `readFile()` results in another asynchronous
 While this works fairly well, you can quickly get into a pattern that has come to be known as *callback hell*. Callback hell occurs when you nest too many callbacks:
 
 ```js
-method1(function(err, result) {
+function1(function(err, result) {
```

Comment:
Not a fan of this change. It's visually hard to parse. I get the point, that these aren't methods, but I don't like this naming.